app/presenters/queries/expanded_link_set.rb

```diff
@@ -61,7 +61,7 @@ def present_expanded_link(link_hash)
           end
 
           if hash[:details]
-            hash[:details] = Presenters::DetailsPresenter.new(hash[:details], nil).details
+            hash[:details] = Presenters::DetailsPresenter.new(hash[:details], nil, nil).details
```

Comment:
I guess this means that we won't expand embed references inside linked `body` elements, so we might end up with things like:

```ruby
{
  content_type: "person",
  # ...
  links: {
    role: [{
      content_type: "role",
      # ...
      details: {
        body: "Some text where we haven't expanded an embed: {{embed:contact:00000000-0000-0000-0000-000000000000}}"
        # ...
      }
    }]
  }
}
```

I think that's fine for now, maybe we'll revisit it later.